code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -349,7 +350,7 @@
 	for(var/obj/machinery/door/D in GLOB.airlocks)
 		if(!is_station_level(D.z))
 			continue
-		INVOKE_ASYNC(D, /obj/machinery/door.proc/hostile_lockdown, src)
+		INVOKE_ASYNC(D, /obj/machinery/door.proc/hostile_lockdown, owner)

```

Comment:
Having src instead of owner here gave me runtimes on every single door that ability affected. I didn't touch this ability so these runtimes might be present on live right now